src/AccessibilityInsights.SharedUx/FileBug/IEBrowserEmulation.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Desktop.Telemetry;
+using Microsoft.Win32;
+using System;
+using System.Diagnostics;
+
+namespace AccessibilityInsights.SharedUx.FileBug
+{
+    /// <summary>
+    /// This class sets several registry values in order to ensure that the hosted WebBrowser control renders
+    /// documents in standards mode. Without these changes the hosted WebBrowser is unable to properly
+    /// render some pages when filing issues
+    /// https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/general-info/ee330730%28v%3dvs.85%29#browser-emulation
+    /// http://msdn.microsoft.com/en-us/library/ee330720(v=vs.85).aspx
+    /// </summary>
+    public static class IEBrowserEmulation
+    {
+        public static void SetFeatureControls()
```

Comment:
Don't we need to clean up after all?

Reply:
These feature controls are per process so we are OK